docs/providers/aws/cli-reference/invoke.md

```diff
@@ -31,6 +31,21 @@ Default is `false`.
 ## Provided lifecycle events
 - `invoke:invoke`
 
+
```

Comment:
Could you extend the general `invoke` definition at the top so that it shows that `local` is an optional parameter (like it's done here: https://serverless.com/framework/docs/providers/aws/cli-reference/deploy/) --> "serverless invoke [local]".